packages/types/src/sszTypes.ts

```diff
@@ -1,13 +1,192 @@
+    SignedBuilderBid: bellatrix.SignedBuilderBid,
+    SSEPayloadAttributes: bellatrix.SSEPayloadAttributes,
+  },
+  [ForkName.capella]: {
+    BeaconBlock: capella.BeaconBlock,
+    BeaconBlockBody: capella.BeaconBlockBody,
+    BeaconState: capella.BeaconState,
+    SignedBeaconBlock: capella.SignedBeaconBlock,
+    Metadata: altair.Metadata,
+    LightClientHeader: capella.LightClientHeader,
+    LightClientBootstrap: capella.LightClientBootstrap,
+    LightClientUpdate: capella.LightClientUpdate,
+    LightClientFinalityUpdate: capella.LightClientFinalityUpdate,
+    LightClientOptimisticUpdate: capella.LightClientOptimisticUpdate,
+    LightClientStore: capella.LightClientStore,
+    BlindedBeaconBlock: capella.BlindedBeaconBlock,
+    BlindedBeaconBlockBody: capella.BlindedBeaconBlockBody,
+    SignedBlindedBeaconBlock: capella.SignedBlindedBeaconBlock,
+    ExecutionPayload: capella.ExecutionPayload,
+    ExecutionPayloadHeader: capella.ExecutionPayloadHeader,
+    BuilderBid: capella.BuilderBid,
+    SignedBuilderBid: capella.SignedBuilderBid,
+    SSEPayloadAttributes: capella.SSEPayloadAttributes,
+  },
+  [ForkName.deneb]: {
+    BeaconBlock: deneb.BeaconBlock,
+    BeaconBlockBody: deneb.BeaconBlockBody,
+    BeaconState: deneb.BeaconState,
+    SignedBeaconBlock: deneb.SignedBeaconBlock,
+    Metadata: altair.Metadata,
+    LightClientHeader: deneb.LightClientHeader,
+    LightClientBootstrap: deneb.LightClientBootstrap,
+    LightClientUpdate: deneb.LightClientUpdate,
+    LightClientFinalityUpdate: deneb.LightClientFinalityUpdate,
+    LightClientOptimisticUpdate: deneb.LightClientOptimisticUpdate,
+    LightClientStore: deneb.LightClientStore,
+    BlindedBeaconBlock: deneb.BlindedBeaconBlock,
+    BlindedBeaconBlockBody: deneb.BlindedBeaconBlockBody,
+    SignedBlindedBeaconBlock: deneb.SignedBlindedBeaconBlock,
+    ExecutionPayload: deneb.ExecutionPayload,
+    ExecutionPayloadHeader: deneb.ExecutionPayloadHeader,
+    BuilderBid: deneb.BuilderBid,
+    SignedBuilderBid: deneb.SignedBuilderBid,
+    SSEPayloadAttributes: deneb.SSEPayloadAttributes,
+    ExecutionPayloadAndBlobsBundle: deneb.ExecutionPayloadAndBlobsBundle,
+  },
+};
+
+const pick = <T extends Record<ForkName, unknown>, K extends keyof T>(obj: T, ...keys: K[]): Pick<T, K> =>
+  Object.fromEntries(keys.filter((key) => key in obj).map((key) => [key, obj[key]])) as Pick<T, K>;
+
+const executionForks: ForkExecution[] = [ForkName.bellatrix, ForkName.capella, ForkName.deneb];
+const lightCLientForks: ForkLightClient[] = [ForkName.altair, ForkName.bellatrix, ForkName.capella, ForkName.deneb];
+const blobsForks: ForkBlobs[] = [ForkName.deneb];
+
+export const allForksExecution = pick(typesByFork, ...executionForks);
+export const allForksLightClient = pick(typesByFork, ...lightCLientForks);
+export const allForksBlobs = pick(typesByFork, ...blobsForks);
+export const allForksBlinded = {
+  bellatrix: {
+    BeaconBlockBody: bellatrix.BlindedBeaconBlockBody,
```

Comment:
may be we can just carry Blinded.... keys directly in the typesByFork and refer them everywhere as .Blinded...

so this can also be cleaned up

Reply:
Will cleanup these type objects some helpers in config object in a separate PR.